markdown/manual.md

```diff
@@ -1,220 +1,251 @@
-
-#### Weekly Team Lunch
-* We have lunch together once a week, just for fun and to catch up.
+#### Daily Standup(s)
+To better coordinate organizational or product development efforts, one or more daily standups may be scheduled, each lasting no longer than 15 minutes.
 
-#### Book Club
-We're a small team in an emerging field. Professional development and continuing education is an important part of all of our jobs. We foster team growth by holding a book club once a month. Anyone is welcome to suggest reading material; it's helpful for that person to facilitate discussion by bringing a few questions or observations from the reading to the discussion.
+The purpose of the daily standup is to provide a short summary of what you focused on the previous day, what you are planning on focusing on today, and, critically, identification of any obstacles or blockers that may prevent you from completing a task.
 
-### Tools & Services We Use
-
-#### Teamwork
-For daily status updates, project planning, and task management. If you need another team member to do something, please assign them a task on Teamwork. Tech support issues on America Decoded sites should be logged as tasks under the "State Decoded" project.
-
-#### Slack
-For team chat.
-
-#### UberConference
-For group calls. Please enter calls on the Phone Conference calendar to avoid double-booking the line.
+Once each participant has completed their summary, the standup may be ended. Should a topic be raised that requires more in depth discussion, a separate meeting should be scheduled.
 
-#### Google Hangouts
-For group video chats. Please append a Google Hangout link to any calendar invite you create and share.
+#### Weekly All Hands
+We meet once a week as a complete team for no more than 30 minutes to discuss organization-level topics, share important updates, and, most importantly, celebrate our work through informal talks or product demos. Should a topic be raised that requires more in depth discussion, a separate meeting should be scheduled.
 
-#### Google Calendar
-Please keep your availability updated on your Google Calendar for the organization.
+Participation in the weekly standup is required of all organization employees and must allow for the full participation of all employees regardless of geographic location.
 
-#### Google Docs
-For group collaboration on written materials, call agendas, our budget tracking document, etc. etc.
+#### Book Clubs, "Family" Meals, and Other Informal Rituals
+We are a small, close knit team. To maintain a balanced, healthy, and fun organization, we encourage the formation of informal gatherings and groups. Any employee may sponsor an informal gathering or group so as long it is are open to all employees and supports organizational health.
```

Comment:
We don't really have any of these going right now. I'd be open to starting book club again, or coming up with another idea.

Being a remote team, I think this stuff is extra important. I would personally like to do more if we can!